pendingtxs/pending_txs.go

```diff
@@ -136,7 +136,7 @@ func validTxWithHighestFee(txs map[types.TransactionID]nanoTx, balance uint64) t
 	bestID := types.EmptyTransactionID
 	var maxFee uint64
 	for id, tx := range txs {
-		if (tx.Fee > maxFee || (tx.Fee == maxFee && bytes.Compare(id[:], bestID[:]) < 0)) &&
+		if (tx.Fee > maxFee || (tx.Fee == maxFee && bytes.Compare(id[:], bestID[:]) < 0) || (tx.Fee == 0 && maxFee == 0)) &&
```

Comment:
i think this will be clearer to rewrite to
```
if balance >= (tx.Amount+tx.Fee) && 
(bestID == types.EmptyTransactionID || // first tx with enough balance
 tx.Fee > maxFee || 
 (tx.Fee == maxFee && bytes.Compare(id[:], bestID[:]) < 0))
```

Reply:
thanks